src/inject/dynamic-theme/index.ts

```diff
@@ -182,7 +193,7 @@ function createDynamicStyleOverrides() {
         .filter((details) => details && details.variables.size > 0)
         .map(({variables}) => variables);
     if (newVariables.length === 0) {
-        styleManagers.forEach((manager) => manager.render(filter, variables, getIgnoreImageAnalysisSelectors()));
+        styleManagers.forEach((manager) => manager.render(filter, legacyVaribales, getIgnoreImageAnalysisSelectors()));
```

Comment:
`legacyVaribales` must be `legacyVariables`

Reply:
👀 Does vscode have some spellchecking?